tuned/plugins/plugin_scheduler.py

```diff
@@ -336,8 +336,8 @@ def _convert_affinity(self, str_affinity):
 		else:
 			affinity = self._cmd.hex2cpulist(str_affinity)
 			if not affinity:
-				log.error("Invalid affinity: %s. It will be ignored."
-						% str_affinity)
+				log.warn("Invalid affinity: %s. It will be ignored."
+						 % str_affinity)
```

Comment:
This can also happen only due to misconfiguration, so I think error is appropriate here.